samples/gen-go/server/router.go

```diff
@@ -50,18 +52,35 @@ func (s *Server) Serve() error {
 		s.l.Info("please provide a kvconfig.yml file to enable app log routing")
 	}
 
-	if lightstepToken := os.Getenv("LIGHTSTEP_ACCESS_TOKEN"); lightstepToken != "" {
-		tags := make(map[string]interface{})
-		tags[lightstep.ComponentNameKey] = "swagger-test"
-		lightstepTracer := lightstep.NewTracer(lightstep.Options{
-			AccessToken: lightstepToken,
-			Tags:        tags,
-			UseGRPC:     true,
-		})
-		defer lightstep.FlushLightStepTracer(lightstepTracer)
-		opentracing.InitGlobalTracer(lightstepTracer)
+	if signalfxToken := os.Getenv("SIGNALFX_ACCESS_TOKEN"); signalfxToken != "" {
+		ingestUrl := os.Getenv("SIGNALFX_INGEST_URL")
+		if ingestUrl == "" {
+			ingestUrl = "https://ingest.signalfx.com"
+		}
+
+		// Create a Jaeger HTTP Thrift transport
+		transport := transport.NewHTTPTransport(ingestUrl+"/v1/trace",
+			transport.HTTPBasicAuth("auth", signalfxToken))
+
+		cfg, err := jaegercfg.FromEnv()
+		if err != nil {
+			log.Fatal("Could not parse Jaeger env vars: ", err.Error())
+		}
+		cfg.ServiceName = "swagger-test"
+		cfg.Sampler = &jaegercfg.SamplerConfig{
+			Type:  jaeger.SamplerTypeRateLimiting,
+			Param: 500,
```

Comment:
comment about what this config means? I had to look up the docs to understand this

Reply:
also we might want to set `MaxOperations` to something (even if it's very high) to make sure jaeger doesn't explode the program's memory footprint if a service has an accidentally large number of distinct operations